srcpkgs/snore/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'snore'
+pkgname=snore
+version=0.2
+revision=1
+build_style=gnu-makefile
+short_desc="Sleep with feedback"
+maintainer="wael <40663@protonmail.com>"
+license="MIT"
+homepage="https://github.com/clamiax/snore"
+distfiles="https://github.com/clamiax/${pkgname}/archive/refs/tags/${version}.tar.gz"
```

Comment:
please expand pkgname

Reply:
someone has told me to do this before with an package before, but why does this exactly need to be happened?

expanding pkg name sure, but what about version?

Reply:
$version should still be a variable because it changes with every update. Many maintainers prefer not to have `$pkgname` and other variables in `$distfiles` for various reasons - one is to make grep easier. $pkgname doesn't usually even save many characters and it doesn't change. It's also annoying to replace all $pkgname references in $distfiles when renaming the package for my own purposes.

Reply:
> $pkgname doesn't usually even save many characters and it doesn't change.

so really, `$pkgname` should be used in the distfiles when the actual package name is bigger than `$pkgname`

Reply:
> so really, $pkgname should be used in the distfiles when the actual package name is bigger than $pkgname

No, others part of that comment are more important.

The thing is: assume there is already pkgname called `snore` in tree, and you want to add https://github.com/clamiax/snore as a second one. Then you need to use other name, e.g. `snore-a-chaty-sleep`. You wouldn't still set `distfiles="https://github.com/clamiax/${pkgname}/archive/refs/tags/${version}.tar.gz"`, right?